getting-started/getting-started.md

```diff
@@ -1,27 +1,1209 @@
+
+To use this generator, open a terminal, navigate to a directory where you have rights to create files, and type:
+
+    sails new --template jade blog
+
+This will create a Sails application called Blog in a blog directory and install the npm dependencies that are already mentioned in `package.json` using `npm install`.
+
+You can see all of the command line options that the Sails application builder accepts by running `sails new -h`.
+
+After you create the blog application, switch to its folder:
+
+    cd blog
+
+The `blog` directory has a number of auto-generated files and folders that make up the structure of a Sails application. The anatomy thereof is described [here](http://sailsjs.org/#/documentation/anatomy/myApp "Anatomy of a SailJs app").
+
+# Hello, Sails!
+
+To begin with, let's get some text up on screen quickly. To do this, you need to get your Sails application server running.
+
+## Starting up the Web Server
+
+You actually have a functional Sails application already. To see it, you need to start a web server on your development machine. You can do this by running the following in the blog directory:
+
+    sails lift
+
+At this point, if you visit ([http://localhost:1337](http://localhost:1337)) you will see the default home page.
+
+*To stop the web server, hit Ctrl+C in the terminal window where it's running. To verify the server has stopped you should see your command prompt cursor again. For most UNIX-like systems including Mac OS X this will be a dollar sign $.*
+
+## Say "Hello", Sails
+
+To get Sails saying "Hello", you need to create at minimum a [controller](http://sailsjs.org/#/documentation/concepts/Controllers/) and a [view](http://sailsjs.org/#/documentation/concepts/Views).
+
+A controller's purpose is to receive specific requests for the application. Routing decides which controller receives which requests. Often, there is more than one route to each controller, and different routes can be served by different actions. Each action's purpose is to collect information to provide it to a view.
+
+A view's purpose is to display this information in a human readable format. An important distinction to make is that it is the controller, not the view, where information is collected. The view should just display that information. By default, view templates are written in a templating language called [Jade](http://jade-lang.com "jade - Node template engine") which is processed by the request cycle in Sails before being sent to the user.
```

Comment:
Hi @rachaelshaw

I believe you're talking about this line? I was referring to the views in this tutorial being written in Jade. Should I change the wording to state that?

P.S I chose Jade because I ran into problems when writing the tutorial in EJS. Not everything I wanted to do was possible (or as easily as with Jade) and the [documentation for EJS](https://code.google.com/p/embeddedjavascript/w/list) is really old (2010).